README.md

```diff
@@ -1,5 +1,27 @@
 # FoodAdvisor Quick Start
 
+FoodAdvisor is a sample application to show you what can be made with [Strapi](https://strapi.io/). This is a restaurant based application, we use the word “food” a lot, because food is sharing and sharing is caring
+
+<div align="center">
+<img src="https://i.ibb.co/d4Ss8sn/foodadvisor.png">
+ </div>
+
+Strapi is a collaborative open-source headless CMS fueled by a community of amazing developers, designers and users. Strapi is more than a Node.js Framework and more than a Headless CMS.
+
+This [application](https://foodadvisor.strapi.io/) allows you preview the different types of foods various restaurants offer. you can see reviews from a particular food/restaurant, restaurant location and restaurant phone number. The goal is to make suggestions and help you pick good food that you like.
+
+## Project Set Up
+
+This application currently uses SQLite as database. To set up and run this project locally on your machine, you'll need the following prerequisites:
+
+- **[Node.js](https://nodejs.org/en/)** Installed on your working environment. The current recommended version to run Strapi is Node v14, a minimum version of 12.x can run Strapi but it isn't recommended.
+
+- **[Yarn](https://classic.yarnpkg.com/en/docs/install/#windows-stable)**, you can install it with NPM by running the command below in your command line
+
+```
+npm i yarn
```

Comment:
Not the correct command to install yarn, it needs to be done globally. Would forward users to yarn install docs instead of trying to replicate it here.